Don't use lock file in Docker setup

tchajed opened the pull request:
Gemfile.lock is git ignored so it isn't in the repo. It'll be generated on first build by bundler.

RalfJung commented:
Maybe we should have it in the repo instead, to actually record which version of things should work?

tchajed commented:
Sure, that is the better solution. I extracted this one from the Docker image after doing a build.

In case anybody wants to use Docker to get Gemfile.lock again, I extracted it from the image with:
```sh
docker build -t iris-project .
docker run --rm -it iris-project cat Gemfile.lock > Gemfile.lock
dos2unix Gemfile.lock
```

RalfJung commented:
Thanks!
